### DISCUSSION TITLE
FileNotFoundError: Cannot find scripts/deploy.py

### DISCUSSION THREAD

**Discussion by DanielHarbours** [smartcontractkit/full-blockchain-solidity-course-py#1752] [Q&A] [answered]
I have managed to compile the contract but when I do to deploy it brownie gives this error:

FileNotFoundError: Cannot find scripts/deploy.py

Earlier on it says this too:

"/Users/NAME/.local/pipx/venvs/eth-brownie/lib/python3.10/site-packages/brownie/project/main.py:743: BrownieEnvironmentWarning: Loaded project has a root folder of '/Users/NAME/demos/brownie_fundme' which is different from the current working directory
warnings.warn(
BrownieFundmeProject is the active project."

So im guessing the project has a seperate root folder from the CWD. But I cant find how to address this. Can anyone help

**Comment by cromewar** [comment, marked as answer]
Hello, it seems you are no using the correct location at running the code on your console.
Be sure to execute the code on the root project folder, remember you can navigate between folders using `cd`.
Also please share a screen shot about how your project (folder and file) structure looks like.

**Reply by DanielHarbours** [reply]
Thanks. This is what my folder structure looks like. 
<img width="1440" alt="Screenshot 2022-10-19 at 10 44 03" src="https://user-images.githubusercontent.com/46790647/196642992-d258781d-1b3d-46df-b5bc-b5143b97eb1a.png">

It makes sense that im not executing the code in the root project folder but Im not sure how to do that?

Like how do I move the project into the root folder?

**Reply by cromewar** [reply]
Well actually I meant you folder structure on `vscode`, however I can see you are using Mac, so you can navigate on the terminal with the following commands:
`ls`: shows the folders and files you are on your current location, by default you will start on your home path:
![image](https://user-images.githubusercontent.com/20199180/196710396-1fae4f85-8526-40a3-9c2f-a6a13e9eff94.png)
Then you can start moving using `cd`, in your case the command should be something like this:
`cd demos/brownie_fundme` or the project name.

However, I have some recommendations for you:
1. Try not to use `pipx` for brownie, is preferably to use regular `pip` to install it.
2. The best python version for brownie right now is 3.8, as 3.10 still has some incompatibilities.
3. Try to use the integrated `vscode` terminal which will open on the project root folder.
4. Be sure again the file structure in `vscode` is correct, I mean that the contract are on the contracts folder, python scripts on it's respective folder and so on.

I hope any of this information might help.

**Comment by DanielHarbours** [comment]
So i tried to unistall brownie so I could re install it. When I try to uninstall it says it isnt there. 

But also says that its in my path. Is that something to do with it? 

Should I leave it in the PATH or move it. And if so how do I move it. Thanks

MacBook-Air-2 ~ % pip uninstall eth-brownie 
WARNING: Skipping eth-brownie as it is not installed.
-MacBook-Air-2 ~ % pip uninstall -brownie 

Usage:   
  pip uninstall [options] <package> ...
  pip uninstall [options] -r <requirements file> ...

no such option: -b
danielharbuz@Daniels-MacBook-Air-2 ~ % pip uninstall brownie
WARNING: Skipping brownie as it is not installed.
MacBook-Air-2 ~ % pipx uninstall brownie
Nothing to uninstall for brownie 😴
⚠️  Note: '/Users/.local/bin/brownie' still exists on your system and is on your PATH
-MacBook-Air-2 ~ % pipx install brownie

No apps associated with package brownie or its dependencies. If you are attempting to install a library, pipx should not be used.
Consider using pip or a similar tool instead.
-MacBook-Air-2 ~ %

**Reply by cromewar** [reply]
pip uninstall -brownie you should not use `-` just brownie.
